test/e2e/framework.go

```diff
@@ -200,6 +214,14 @@ func (f *Framework) afterEach() {
 	f.Client = nil
 }
 
+func (f *Framework) CreateNamespace(baseName string, c *client.Client, labels map[string]string) (*api.Namespace, error) {
+	ns, err := createTestingNS(baseName, c, labels)
```

Comment:
The one remaining use of `createTestingNS` is `test/e2e/namespace.go`, which *should* be using Framework. (Everything should be using Framework.) But we can punt that, I guess. sigh.